alws/routers/releases.py

```diff
@@ -120,3 +120,45 @@ async def revert_db_release(
         )
     revert_release.send(release_id, user.id)
     return {"message": "Release plan revert has been started"}
+
+
+@router.delete(
+    '/{release_id}/delete',
+    response_model=release_schema.ReleaseCommitResult,
+)
+async def delete_release(
+    release_id: int,
+    db: AsyncSession = Depends(get_db),
+    user: models.User = Depends(get_current_user),
+):
+    """
+    Delete only a scheduled release
+    """
+
+    async with db.begin():
+        release = (
+            (
+                await db.execute(
+                    select(models.Release).where(
+                        models.Release.id == release_id,
+                        models.Release.status == ReleaseStatus.SCHEDULED,
+                    )
+                )
+            )
+            .scalars()
+            .first()
+        )
+        if release is None:
+            return {
+                'message': (
+                    'There is no scheduled release plant with ID '
+                    f'"{release_id}"'
+                ),
+            }
+        else:
+            await db.delete(release)
+            return {
+                'message': (
+                    f'Scheduled release with ID "{release_id}" is removed'
+                ),
+            }
```

Comment:
I'm not sure that we need this endpoint

Reply:
This is more for developer for now, but actually Eduard already asked for ability to delete releases when they are not started yet